docs/authentication.md

```diff
@@ -0,0 +1,47 @@
+# Authentication
+PrairieLearn currently has a few ways to do user authentication. 
+- [Google OAuth 2](#google-oauth-2)
+- [Azure](#azure)
+- [Shibboleth](#shibboleth)
+- [LTI](./courseInstance.md#lti-overview)
+
+
+## Google OAuth 2
+> Take a look in `pages/authLoginOAuth2/authLoginOAuth2.js` and `pages/authCallbackOAuth2/authCallbackOAuth2.js` to get an idea of what variables you need to modify.

```

Comment:
This isn't semantically a block quote; I'd leave this as plain text. Ditto for elsewhere.

Reply:
Done